custom_components/aqara_gateway/binary_sensor.py

```diff
@@ -714,9 +714,11 @@ def update(self, data: dict = None):
                 'entity_id': self.entity_id, 'click_type': self._state
             })
 
-            time.sleep(.1)
-
-            self._state = ''
+            # reset the state to empty after 0.1 second
+            if self.device['model'] in ('lumi.vibration.aq1'):
+                self.hass.loop.call_later(.1, self.reset_state('unknown'))
```

Comment:
At least there is a syntax error here.

Reply:
Hi, I run this code on my HA i don't seem to get any syntax error on my side can you kindly point it out ? or is it a coding style that i'm not matching ? Thanks